src/test/regress/sql/shard_rebalancer.sql

```diff
@@ -1427,3 +1428,17 @@ RESET client_min_messages;
 
 DROP TABLE "events.Energy Added", colocated_t1, colocated_t2, colocated_t3;
 RESET citus.shard_count;
+DROP VIEW table_placements_per_node;
+DELETE FROM pg_catalog.pg_dist_rebalance_strategy WHERE name='capacity_high_worker_2';
+DELETE FROM pg_catalog.pg_dist_rebalance_strategy WHERE name='only_worker_1';
+
+\c - - - :worker_1_port
+SET citus.enable_ddl_propagation TO OFF;
+REVOKE ALL ON SCHEMA public FROM testrole;
+DROP USER testrole;
+
+\c - - - :worker_2_port
+SET citus.enable_ddl_propagation TO OFF;
+REVOKE ALL ON SCHEMA public FROM testrole;
+DROP USER testrole;
+DROP TABLE test_rebalance_with_disabled_worker_433500, test_rebalance_with_disabled_worker_433501, test_rebalance_with_disabled_worker_433502, test_rebalance_with_disabled_worker_433503;
```

Comment:
Is it expected that these were not removed?

Reply:
yes, they are left on a disabled node. We didn't notice it until this pr. Now we get errors because of them, in the flaky check

Reply:
The main thing I'm wondering is if this is intended or if this is a bug. No need to fix it in this PR, but if it's a bug we should at least open an issue.